src/EditorFeatures/CSharp/DecompiledSource/DocCommentConverter.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.DocumentationComments;
+using Microsoft.CodeAnalysis.MetadataAsSource;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.DecompiledSource
+{
+    // This is a copy of CSharpMetadataAsSourceService.DocCommentConverter
```

Comment:
I know that this is bad, because it essentially is code duplication, but I am not sure whether it's possible to make the `CSharpMetadataAsSourceService.DocCommentConverter` accessible for the new `CSharpDecompiledSourceService`.

Reply:
I moved it to `Microsoft.CodeAnalysis.CSharp.DocumentationComments` in MS.CA.CS.Features. I hope that's a suitable location.